src/index.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-restricted-syntax */
+import './style.css';
+import { getTaskLS } from './interactive.js';
+import addtodoItem, { manipulate, clearCompleted } from './addRemove.js';
+
+const toDoList = document.querySelector('.toDoList');
+const toDoBtn = document.querySelector('.btnToDo');
+const clearBtn = document.querySelector('.btn-clear');
+
+toDoList.addEventListener('click', manipulate);
+toDoBtn.addEventListener('click', addtodoItem);
+clearBtn.addEventListener('click', clearCompleted);
+
+function loadTodo() {
+  const toDoObjects = getTaskLS();
```

Comment:
As you are using ES6, It will be better to use arrow function instead of normal function because arrow function is shorter and simpler than normal function

Reply:
I have noted that arrow functions are shorter and will make use of them in future projects. Because of the deadline, I opted to use a method I am familiar with and will easily debug if I have any issues.